wrench/src/yaml_frame_writer.rs

```diff
@@ -324,40 +324,50 @@ impl YamlFrameWriter {
         let bytes = data.bytes.take().unwrap();
         let (path_file, path) = Self::next_rsrc_paths(&self.rsrc_prefix, &mut self.next_rsrc_num, &self.rsrc_base, "img", "png");
 
-        let ok = match data.format {
+        // takes a buffer with a stride and copies it into a new buffer that has stride == width
+        fn unstride<T: Clone>(mut slice: &[T], width: usize, stride: usize) -> Vec<T> {
```

Comment:
Since I like converting everything to iterators, as long as it makes sense, and this function does make some, I got this:
```rust
    slice.chunks(stride)
         .flat_map(|chunk| chunk[..width].iter().cloned())
         .collect()
```